### ISSUE TITLE
[css-zoom?] Zoom and CSSOM

### ISSUE THREAD

**Issue by w3cbot** [w3c/a11y-review#183]
**This is a tracker issue.** Only discuss things here if they are a11y group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/csswg-drafts/issues/9398

**Comment by AutoSponge** [comment]
This doesn't change the advice (e.g., WCAG Focus Not Obscured). It gives developers a predictable model for understanding the layout of elements using CSS zoom which should benefit all.
